devicetypes/tommysqueak/danfoss-living-connect-radiator-thermostat.src/danfoss-living-connect-radiator-thermostat.groovy

```diff
@@ -13,17 +13,18 @@
  *  for the specific language governing permissions and limitations under the License.
  *
  */
+
 metadata {
-	definition (name: "Danfoss Living Connect Radiator Thermostat LC-13 v3", namespace: "tommysqueak", author: "Tom Philip") {
+	definition (name: "Danfoss Living Connect Radiator Thermostat LC-13 v3.01", namespace: "tommysqueak", author: "Tom Philip") {
 		capability "Actuator"
 		capability "Sensor"
 		capability "Thermostat"
 		capability "Battery"
 		capability "Configuration"
 		capability "Switch"
 
-		command "temperatureUp"
-		command "temperatureDown"
+		command "tempUp"
```

Comment:
on http://docs.smartthings.com/en/latest/device-type-developers-guide/tiles-metadata.html#thermostat-multi-attribute-tile as default they use tempUp / tempDown so I decided that I should do it according to the docs, greater transparency for the less experienced